core/src/components/toast/toast.scss

```
@@ -120,6 +120,15 @@
   justify-content: center;
 }
 
+.toast-icon {
+  /* stylelint-disable property-blacklist */
+  margin-left: 15px;
+
+  padding: 10px;
+   /* stylelint-enable property-blacklist */
+
+  font-size: 1.4em;
```

```suggestion
  @include margin(0, 0, 0, 15px);
  @include padding(10px, 10px, 10px, 10px);
```

We need to include the mixins here to account for RTL mode. Currently, the styles are not consistent between LTR and RTL modes, but the mixins will handle this switch for us.

Also, no need to add the font size here as it should apply in the same block as `.toast-button-icon` (see my other comment)